### ISSUE TITLE
Howcome Alphawallet is not listed on F-droid?

### ISSUE THREAD

**Issue by re42** [AlphaWallet/alpha-wallet-android#1573]
Would you be able to add this on F-droid please? Some users arent comfortable downloading from playstore.

**Comment by colourful-land** [comment]
That's a good idea. Need to figure out how to submit it there. @AW-STJ might we look into that together?

**Comment by abbluiz** [comment]
+1 submit on F-Droid

@jecogeo

**Comment by easyKL** [comment]
Request for packing done:
https://gitlab.com/fdroid/rfp/-/issues/1633
Please consider a build with Firebase striped (and others Non-Free dependencies)
https://f-droid.org/docs/Inclusion_Policy/

**Comment by fossdd** [comment]
Yes, you're right. We won't package a app based on properitary libraries, that we can't build self.
There is an option, for an version without Crashlytics and these stuff. But maybe switch to another Crash reporting library, like ACRA. F-Droid is also using ACRA.
If you still want have these propertiary dependencies, you can do a own repository for F-Droid. See here the docs: https://f-droid.org/en/docs/Setup_an_F-Droid_App_Repo/

**Comment by IzzySoft** [comment]
@colourful-land F-Droid maintainer here. I'm not an Android dev, so I do not fully understand your implementation. It seems you're already using build flavors, and even have a `noAnalytics` flavor – but then, Crashlytics is integrated using

```gradle
 implementation 'com.google.firebase:firebase-crashlytics:17.3.1'
```

where I'd have expected something like `analyticsImplementation`. See my [full comment here](https://gitlab.com/fdroid/rfp/-/issues/1633#note_499651242) – corrections and explanations welcome. If you could provide an APK that was built using the `noAnalytics` flavor, I gladly check it for you for its "contents".

**Comment by FestplattenSchnitzel** [comment]
Hello,

currently we cannot get https://gitlab.com/fdroid/rfp/-/issues/1633 done, as this issue here isn't done yet.

If there is no activity here or in the F-Droid issue in the next 4 weeks, we will close our issue to reduce the number of open issues.
In our [wiki](https://gitlab.com/fdroid/wiki/-/wikis/Internal/Issue-Cleanup-Policy) you can see more details about this process.

Please contact us when you need help.

Thank you for helping us spreading FOSS apps in the world :)

**Comment by colourfreak** [comment]
@hboon Any progress on this one? Kind reminder.

**Comment by JamesSmartCell** [comment]
> @hboon Any progress on this one? Kind reminder.

There was some outstanding preparation work we needed to do to get it listed, which is now complete. I have re-submitted the inclusion issue here:

https://gitlab.com/fdroid/rfp/-/issues/1916

**Comment by colourfreak** [comment]
Nice, can we try here as well? https://github.com/AlphaWallet/alpha-wallet-android/issues/1215

**Comment by colourfreak** [comment]
@hboon Shall we transfer this issue to AlphaWallet-web3-provider repo?

**Comment by colourfreak** [comment]
@seabornlee @justindg Can you take care of this issue?

**Comment by seabornlee** [comment]
[MR](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/10793) created, waiting for merge.

**Comment by hboon** [comment]
Out of phase 1

**Comment by 48-mq** [comment]
Team, any update on this?

**Comment by 48-mq** [comment]
@JamesSmartCell @micwallace ^
